packages/rollup-contracts/contracts/FullStateManager.sol

```diff
@@ -3,17 +3,38 @@ pragma experimental ABIEncoderV2;
 
 /* Internal Imports */
 import {StateManager} from "./StateManager.sol";
+import {SafetyChecker} from "./SafetyChecker.sol";
 
 /**
  * @title FullStateManager
  * @notice The FullStateManager is used for off-chain tx evaluation. It holds a complete mapping
  *         of all chain storage.
  */
 contract FullStateManager is StateManager {
+    // Add Safety Checker contract
+    SafetyChecker safetyChecker;
+    // for testing: if true, then do not perform safety checking on init code or deployed bytecode
+    bool overrideSafetyChecker;
+
+    address ZERO_ADDRESS = 0x0000000000000000000000000000000000000000;
+
+    // bitwise right shift 28 * 8 bits so the 4 method ID bytes are in the right-most bytes
```

Comment:
Not sure where this came from, would delete or move if you know off the top where it originally was